tests/daily_scraper/test_daily_scraper.py

```diff
@@ -6,309 +6,132 @@
-     'url': 'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-central/firefox-27.0a1.en-US.linux-x86_64.tar.bz2'
-    },
-
-    # -p mac --branch=mozilla-central
-    {'args': {'platform': 'mac',
-               'branch': 'mozilla-central'},
-    'filename': '2013-10-01-03-02-04-mozilla-central-firefox-27.0a1.en-US.mac.dmg',
-    'url': 'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-central/firefox-27.0a1.en-US.mac.dmg'
-    },
-
-    # -p linux --branch=mozilla-central --extension=txt
-    {'args': {'platform': 'linux',
-              'branch': 'mozilla-central',
-              'extension': 'txt'},
-    'filename': '2013-10-01-03-02-04-mozilla-central-firefox-27.0a1.en-US.linux-i686.txt',
-    'url': 'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-central/firefox-27.0a1.en-US.linux-i686.txt'
-    },
-
-    # -p win32 --branch=mozilla-central -l it
-    {'args': {'platform': 'win32',
-              'branch': 'mozilla-central',
-              'locale': 'it'},
-    'filename': '2013-10-01-03-02-04-mozilla-central-firefox-27.0a1.it.win32.installer.exe',
-    'url': 'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-central-l10n/firefox-27.0a1.it.win32.installer.exe'
-    },
-
-    # -p win32 --branch=mozilla-central -l sv-SE
-    {'args': {'platform': 'win32',
-              'branch': 'mozilla-central',
-              'locale': 'sv-SE'},
-    'filename': '2013-10-01-03-02-04-mozilla-central-firefox-27.0a1.sv-SE.win32.installer.exe',
-    'url': 'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-central-l10n/firefox-27.0a1.sv-SE.win32.installer.exe'
-    },
-
-    # -p win32 --branch=mozilla-central --build-id=20130706031213
-    {'args': {'platform': 'win32',
-              'branch': 'mozilla-central',
-              'build_id': '20130706031213'},
-    'filename': '2013-07-06-03-12-13-mozilla-central-firefox-27.0a1.en-US.win32.installer.exe',
-    'url': 'firefox/nightly/2013/07/2013-07-06-03-12-13-mozilla-central/firefox-27.0a1.en-US.win32.installer.exe'
-    },
-
-    # -p win32 --branch=mozilla-central --date=2013-07-02
-    {'args': {'platform': 'win32',
-              'branch': 'mozilla-central',
-              'date': '2013-07-02'},
-    'filename': '2013-07-02-04-12-13-mozilla-central-firefox-27.0a1.en-US.win32.installer.exe',
-    'url': 'firefox/nightly/2013/07/2013-07-02-04-12-13-mozilla-central/firefox-27.0a1.en-US.win32.installer.exe'
-    },
-
-    # -p win32 --branch=mozilla-central --date=2013-07-02 --build-number=1
-    {'args': {'platform': 'win32',
-              'branch': 'mozilla-central',
-              'date': '2013-07-02',
-              'build_number': 1},
-    'filename': '2013-07-02-03-12-13-mozilla-central-firefox-27.0a1.en-US.win32.installer.exe',
-    'url': 'firefox/nightly/2013/07/2013-07-02-03-12-13-mozilla-central/firefox-27.0a1.en-US.win32.installer.exe'
-    },
-
-    # -p win32 --branch=mozilla-central --stub (old format)
```

Comment:
Please keep a comment which mentions that this is the fold format. This also applies to the next test for the new format.

Reply:
It would be enough to say `# Old stub format`.